packages/password-field/src/vaadin-password-field-mixin.js

```diff
@@ -0,0 +1,265 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
+import { FocusMixin } from '@vaadin/a11y-base/src/focus-mixin.js';
+import { SlotController } from '@vaadin/component-base/src/slot-controller.js';
+import { SlotStylesMixin } from '@vaadin/component-base/src/slot-styles-mixin.js';
+import { InputMixin } from '@vaadin/field-base/src/input-mixin.js';
+
+/**
+ * @polymerMixin
+ * @mixes DisabledMixin
+ * @mixes FocusMixin
+ * @mixes InputMixin
+ * @mixes SlotStylesMixin
+ */
+export const PasswordFieldMixin = (superClass) =>
```

Comment:
`dedupingMixin` should be used here?

Reply:
Probably not, since this mixin isn't supposed to be applied multiple times.

Reply:
Ah, nevermind, this mixin is always added once.